### PR TITLE
Adds the SecGear, HyGear, and MasterDrobe vendors to the code and maps.

### PR DESCRIPTION
**Pull request by Firecage**
This PR adds the three new vendors to the code and map(Well, technically 7 due to 5 SecGears)

:cl: Firecage
add: Adds three new vendors. SecGear, HyGear, and MasterDrobe.
/:cl:

hygear_0
hygear-broken_0
hygear-deny_0
hygear-off_0
hygear-panel_0
masterdrobe_0
masterdrobe-broken_0
masterdrobe-deny_0
masterdrobe-off_0
masterdrobe-panel_0
secgear_0
secgear-broken_0
secgear-deny_0
secgear-off_0
secgear-panel_0

### CODE REVIEW

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -28,6 +28,79 @@
...
+/obj/machinery/vending/wardrobe/sec_wardrobe/gear/engineering
+	name = "\improper SecGear Engineering Version"
+	products = list(/obj/item/clothing/suit/armor/vest = 1,
+					/obj/item/clothing/head/helmet/sec = 1,
+					/obj/item/radio/headset/headset_sec = 1,
+					/obj/item/radio/headset/headset_sec/alt = 1,
+					/obj/item/clothing/glasses/hud/security/sunglasses = 1,
+					/obj/item/flashlight/seclite = 1,
+					/obj/item/clothing/accessory/armband/engine = 2,
+					/obj/item/encryptionkey/headset_eng)
```
Comment (ShizCalev) [inline code/modules/vending/wardrobes.dm]: = 1

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/vending/wardrobes.dm [hunk 2]
```diff
@@ -119,8 +192,32 @@
...
+					/obj/item/radio/headset/headset_cargo = 1,
+					/obj/item/clothing/suit/fire/firefighter = 1,
+					/obj/item/clothing/gloves/fingerless = 1,
+					/obj/item/megaphone/cargo = 1,
+					/obj/item/tank/internals/emergency_oxygen = 1,
+					/obj/item/clothing/mask/gas = 1,
+					/obj/item/clothing/head/soft = 1,
+					/obj/item/export_scanner = 1,
+					/obj/item/door_remote/quartermaster = 1,
+					/obj/item/circuitboard/machine/techfab/department/cargo = 1)
```
[the remark this answers is not in the document]
Comment (SpaceManiac) [reply code/modules/vending/wardrobes.dm]: Yeah these aren't really clothing, I thought these items were supposed to be in the QM locker?

**Comment by SpaceManiac** [comment]
Replacing the wardrobe lockers with wardrobe vendors makes some sense - the clothing is replaceable, interchangeable, and most of it won't be used each round - but the personal equipment lockers should stay lockers I think.

**Comment by Firecage** [comment]
Guess that's probably a good enough point SpaceManiac.

**Pull request closed without merging**